target/linux/ramips/image/mt7621.mk

```diff
@@ -942,6 +942,23 @@ define Device/xiaomi_mir3p
 endef
 TARGET_DEVICES += xiaomi_mir3p
 
+define Device/xiaomi_mi-router-ac2100
+  $(Device/uimage-lzma-loader)
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  KERNEL_SIZE := 4096k
+  IMAGE_SIZE := 124416k
```

Comment:
Shouldn't it be 120320k? It's only used by rootfs0.bin.

Reply:
Your sure? I think IMAGE_SIZE will need to contain both ubi and kernel. I'm not %100 sure but there is another PR approved and merged which takes both ubi and kernel at IMAGE_SIZE.

Here it is: https://github.com/openwrt/openwrt/pull/2954

Reply:
The main purpose of IMAGE_SIZE is for `check-size`, and here it is used by rootfs0.bin. (If `check-size` is used for sysupgrade.bin, then IMAGE_SIZE should contain both kernel and rootfs.) If two models share the same partition table and image recipe, then RM2100 should be fixed as well.

Reply:
Probably mans0n is right. We will have to adjust the IMAGE_SIZE to 120320k for R2100 and for RM2100. @emirefek Could you please correct and commit it?

Reply:
Corrected and commited. Thanks @mans0n @phs07